src/components/dashboard/SendQRSummary.js

```diff
@@ -144,8 +145,15 @@ const SendQRSummary = (props: AmountProps) => {
   )
 }
 
-const styles = StyleSheet.create({
-  descriptionText: { maxWidth: 210 },
+const getStylesFromProps = ({ theme }) => ({
+  descriptionText: {
```

Comment:
This is repeating a lot, perhaps we should put this as a property on Section.Text

Reply:
indeed, as we discussed... as a property of `Section.Text`, would be great. We will follow with this approach in #386